bigframes/session/__init__.py

```diff
@@ -1264,13 +1265,45 @@ def read_gbq_function(
             <BLANKLINE>
             [3 rows x 3 columns]
 
+        You can even use a function with multiple inputs. For example, let's use
```

Comment:
Nit from documentation style guide:

> In general, address the reader of your documents using the second person instead of the first person: use you or your instead of we, our, or us.

https://developers.google.com/style/person

Drop the "let's" (the `'s` means "us"):

```suggestion
        You can even use a function with multiple inputs. For example, use
```